html/gui/js/modules/dashboard/ChartComponent.js

```diff
@@ -1,11 +1,11 @@
 /**
- * XDMoD.Modules.SummaryPortlets.ChartPortlet
+ * XDMoD.Module.DashboardComponent.Chart
```

Comment:
```suggestion
 * XDMoD.Module.Dashboard.ChartComponent
```

Personally, I don't find a lot of value in repeating the component name in the header comment, but others may disagree.